.gitignore

```diff
@@ -6,3 +6,4 @@
 packrat/lib*/
 *rsconnect*
 .direnv*/
+inst/scratch/*
```

Comment:
Should we simply ignore any file or directory including `"scratch"`? That seems reasonable to me. If so, we can simply change this line to `scratch`.

Reply:
Fine to leave this as is for now.